modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= Understanding {product-title} upgrade channels
+= {product-title} Upgrade Channels and Update Types
 
-In {product-title} 4.1, Red Hat introduced the concept of upgrade channels for
-recommending the appropriate upgrade versions to your cluster. Upgrade channels
-separate upgrade strategies and also are used to control the cadence of updates.
-Channels are tied to a minor version of {product-title}. For instance,
-{product-title} {product-version} channels will never include an upgrade to a
+In {product-title} 4.1, Red Hat introduced the concept of Upgrade Channels for
+recommending the appropriate upgrade versions to your cluster. Upgrade Channels
+separate upgrade strategies and are also used to control the cadence of updates.
+Upgrade Channels are tied to a minor version of {product-title}. For instance,
+{product-title} {product-version} Upgrade Channels will never include an upgrade to a
```

Comment:
I personally prefer a single "update channels" (or "upgrade channels"), followed by "channels".  But whatever, folks will understand either way.

Reply:
I don't feel strongly either way but what I was attempting to do was identify Upgrade Channel to the reader (customer) as a proper noun and then have it become familiar since Upgrade Channels have become a central and all-important feature of OpenShift and our upgrade strategy.

Reply:
I do not have a personal preference either. However we have to make sure customer/reader does not get confused between the two. So if we have been using `upgrade` word then we should just keep using it.